packages/psnotify.vm/psnotify.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>psnotify.vm</id>
+    <version>0.2.4</version>
+    <authors>alphillips-lab</authors>
+    <description>A POC tool to fight .NET anti-dumping tricks. It does this with two specific techniques: Dump process image to disk on exit Prevent user-initiated garbage collection</description>
```

The two options are together and it is bit difficult to read, what about the following?
```suggestion
    <description>A POC tool to fight .NET anti-dumping tricks. It does this with two specific techniques: Dump process image to disk on exit, prevent user-initiated garbage collection</description>
```

We need a description because it is required, but it doesn't need to include many details (ideally as less details that we don't need to ever update it). So we can also use something like:
```suggestion
    <description>A POC tool to fight .NET anti-dumping tricks./description>
```